study-builder/studies/testboston/i18n/en.conf

```diff
@@ -1,39 +1,155 @@
+    "s2_expect_p1": """
+      Lorem Ipsum.""",
+    "s2_benefits_title": "Why might you choose to take part in this study?",
+    "s2_benefits_p1": """
+      Lorem Ipsum.""",
+
+    "s3_name": "Full Form",
+    "s3_title": "Consent Form",
+    "s3_p1": "Lorem Ipsum. Here is the full form.",
+
+    "s4_name": "Signature",
+    "s4_title": "Informed Consent and Authorization",
+    "s4_self_title": "Statement of Person Giving Informed Consent and Authorization",
+    "s4_self_item1": "I have read this consent form.",
+    "s4_self_item2": """
+      This research study has been explained to me, including risks and possible benefits (if any),
+      other possible treatments or procedures, and other important things about the study.""",
+    "s4_self_item3": "I have had the opportunity to ask questions.",
+    "s4_self_item4": "I understand the information given to me.",
+    "s4_self_agree": """
+      I give my consent to take part in this research study and agree to allow my health information
+      to be used and shared as described above.""",
+    "s4_clinician_title": "Statement of Study Doctor or Person Obtaining Consent",
+    "s4_clinician_item1": "I have explained the research to the study subject.",
+    "s4_clinician_item2": "I have answered all questions about this research study to the best of my ability.",
   },
 
   "covid_survey": {
     "name": "Covid Survey",
-    "title": "Covid Survey",
-    "been_tested_prompt": "Have you ever been previously tested or COVID-19?"
+    "title": "Please tell us about your experience with COVID-19",
+  },
+
+  "address": {
+    "name": "Mailing Address",
+    "title": "Mailing Address",
+    "address_title": "Your Mailing Address"
   },
+
+  "who_filling_prompt": "Who is filling out this form?",
+  "self_sig_prompt": "Signature of Subject",
+  "self_sig_placeholder": "Type your signature here",
+  "clinician_sig_prompt": "Study Doctor or Person Obtaining Consent",
+
+  "covid_been_tested_prompt": "Have you ever been tested for COVID-19?",
+  "covid_test_kind_prompt": "What kind of test(s) did you get? (Check all that apply)",
+  "covid_test_date_prompt": "When did you get tested?",
+  "covid_test_result_prompt": "What was the result of your COVID-19 test?",
+
+  "viral_test_result_prompt": "Swab or saliva test result:",
+  "viral_test_date_prompt": "Swab or saliva test date:",
+  "viral_test_list_add": "+ Add another swab or saliva test",
+
+  "blood_test_result_prompt": "Blood test result:",
+  "blood_test_date_prompt": "Blood test date:",
+  "blood_test_list_add": "+ Add another blood test",
+
+  "current_symptoms_prompt": "Do you currently have any of the following symptoms? (check all that apply)",
+  "past_symptoms_prompt": "In the past 4 weeks, have you had the following symptoms? (check all that apply)",
+
+  "household_diagnosed_prompt": "Has anyone that you live with been tested and diagnosed with COVID-19?",
+  "household_diagnosed_tip": """
```

Comment:
Looks like in Angular SDK we don't have support for anything other than showing the prompt. I'm designating the tooltip as a separate string here, but concatenating it in the actual activity definition.